pkg/controllers/managementuser/certsexpiration/certsexpiration.go

```diff
@@ -35,6 +35,24 @@ type Controller struct {
 }
 
 func Register(ctx context.Context, userContext *config.UserContext) {
+	stater := userContext.DeferredStart(ctx, func(ctx context.Context) error {
```

Comment:
Should this be "starter"?
```suggestion
	starter := userContext.DeferredStart(ctx, func(ctx context.Context) error {
```